packages/components/src/components/post-header/post-header.tsx

```diff
@@ -234,10 +241,15 @@ export class PostHeader {
   }
 
   private renderNavigation() {
-    const navigationClasses = ['navigation'];
+    let navigationClasses = ['navigation'];
     if (this.mobileMenuExtended) {
       navigationClasses.push('extended');
     }
+    if (this.megadropdownOpen) {
+      navigationClasses = navigationClasses.filter(className => className !== 'scroll-y');
+    } else {
+      navigationClasses.push('scroll-y');
+    }
```

Comment:
`navigationClasses` is defined each time the function is called. The string `scroll-y` will never be part of the array because the value is not persisted anywhere.

```suggestion
    if (!this.megadropdownOpen) {
      navigationClasses.push('scroll-y');
    }
```